requirements.txt

```diff
@@ -0,0 +1,4 @@
+ops
```

Comment:
not needed for publishing, but might be nice to have versions or incorporate poetry tool

Reply:
We are in the process of implementing `pip-tools` in order to achieve the same purpose (freezing dependencies). This project doesn't have this yet though.